### DISCUSSION TITLE
emails are not sent

### DISCUSSION THREAD

**Discussion by dveretennikov1** [outline/outline#3284] [Self Hosting]
I'm trying to send invites to new users, got a message in ui that message was sent
![image](https://user-images.githubusercontent.com/102303381/160085077-8564baf1-47c1-4670-a39f-f35ef711c6a3.png)

But there is no any messages in recipients mailboxes
also tried to set in docker.env
DEBUG=emails,mailer

SMTP_HOST="smtp.gmail.com"
SMTP_PORT="465"
SMTP_USERNAME="xxx"
SMTP_PASSWORD="rxxx"
SMTP_FROM_EMAIL="xxx"
SMTP_REPLY_EMAIL=""
SMTP_TLS_CIPHERS="TLSv1.2"
SMTP_SECURE="true"

logs are empty
outline          | {"label":"lifecycle","level":"info","message":"Starting collaboration service"}
outline          | {"label":"lifecycle","level":"info","message":"Starting websockets service"}
outline          | {"label":"lifecycle","level":"info","message":"Starting worker service"}
outline          | {"label":"lifecycle","level":"info","message":"Starting web service"}
outline          | {"level":"warn","message":"Enforced https was disabled with FORCE_HTTPS env variable"}
outline          | {"label":"lifecycle","level":"info","message":"Listening on http://localhost:3000"}

**Comment by YunusEmreAlps** [comment]
Hi,

```
SMTP_HOST=smtp.mail.yahoo.com
SMTP_PORT=587
SMTP_USERNAME=yourmail@yahoo.com
SMTP_PASSWORD=yourapppassword
SMTP_FROM_EMAIL=yourmail@yahoo.com
SMTP_REPLY_EMAIL=yourmail@yahoo.com
SMTP_TLS_CIPHERS=
SMTP_SECURE=false
```

This is my yahoo configuration and working. You can use yahoo SMTP for this project

All the best, 
Have a good day

**Comment by dveretennikov1** [comment]
My problem was the following: 
Outline sent only one invite letter (for testing I use only one user email), if you want to resend invite you should open Member list in Outline, choose show Everyone and delete user

**Reply by YunusEmreAlps** [reply]
I thought you couldn't provide the email link. I did not read your message carefully. Sorry

**Reply by dveretennikov1** [reply]
your answer was helpful too, i made an email on yahoo :)

**Comment by almereyda** [comment]
As Docker Compose does not need the quotation marks `""` to delineate variables, could you try removing those from your `.env` file and see, if that allows you to connect to GMail then?

I have seen cases where the quotation marks were *added* to a secret, and as such it was rendered invalid.

From https://docs.docker.com/compose/env-file/#syntax-rules:

> There is no special handling of quotation marks. This means that they are part of the VAL.
